client/src/client_sync/v17/blockchain.rs

```diff
@@ -297,3 +297,16 @@ macro_rules! impl_client_v17__verifytxoutproof {
         }
     };
 }
+
+/// Implements Bitcoin Core JSON-RPC API method `pruneblockchain`
+#[macro_export]
+macro_rules! impl_client_v17__pruneblockchain {
+    () => {
+        impl Client {
+            /// Instructs the node to prune the blockchain up to a specified height or timestamp.
+            pub fn prune_blockchain(&self, target: u64) -> Result<PruneBlockchain> {
+                self.call("pruneblockchain", &[into_json(target)?])
```

Comment:
FYI for std types we can just call `into()`. `into_json` is for using `serde` serialization on `rust-bitcoin` types.

```suggestion
                self.call("pruneblockchain", &[target.into())
```

Reply:
True, thank you! I have resolved it.